src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBArgumentCollection.java

```diff
@@ -8,11 +8,11 @@
 
 public class GenomicsDBArgumentCollection implements Serializable {
   private static final long serialVersionUID = 1L;
-  public static final String USE_VCF_CODEC_LONG_NAME = "genomicsdb-use-vcf-codec";
+  public static final String USE_BCF_CODEC_LONG_NAME = "genomicsdb-use-bcf-codec";
```

Comment:
I notice that this argument does not control which codec is used in `GenomicsDBImport.getReaderFromPath()`, which is hardcoded to use a `VCFCodec`. Should this argument affect the codec used in `GenomicsDBImport`?

Reply:
This does not apply to GenomicsDBImport as `VCFCodec` is hardcoded there - see [GenomicsDBImport.java#L833](https://github.com/broadinstitute/gatk/blob/d7840b93b3131498c2d1769118fa1d3dd02b7b56/src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java#L833). I am guessing this is because the `--variant/-V` option for GenomicsDBImport mentions only vcf files as inputs. However, GenomicsDB itself can accept either vcf/bcf files and if we allow bcf files as inputs in the future, it will be the tool deciding the Codec and not the user.